virttest/backends/v2v/cfg/convert_source.cfg

```diff
@@ -44,3 +44,10 @@ variants:
                 vpx_dc = VPX_80_DC_V2V_EXAMPLE
                 esx_hostname = ESX_80_HOSTNAME_V2V_EXAMPLE
                 vddk_libdir_src = MOUNT_SRC_VDDK80_LIB_DIR_V2V_EXAMPLE
+            #since vcenter9 is not ready yet, keep using vcenter8 for vddk9.0.0 testing 
+            - esx_90:
+                datastore = VPX_80_DATASTORE_V2V_EXAMPLE
```

Comment:
You can name them to XXX_90_XXXX and set the values to 80 in ci_share_data project.